doc/topics/development/contributing.rst

```diff
@@ -219,201 +219,34 @@ Fork a Repo Guide_>`_ and is well worth reading.
-behavior means that changes submitted to older main release branches will
-automatically be "merged-forward" into the newer branches.
-
-For example, a pull request is merged into ``2017.7``. Then, the entire
-``2017.7`` branch is merged-forward into the ``2018.3`` branch, and the
-``2018.3`` branch is merged-forward into the ``|repo_primary_branch|`` branch.
-
-This process makes is easy for contributors to make only one pull-request
-against an older branch, but allows the change to propagate to all **main**
-release branches.
-
-The merge-forward work-flow applies to all main release branches and the
-operation runs continuously.
-
-Merge-Forwards for Dot Release Branches
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The merge-forward policy applies to dot release branches as well, but has a
-slightly different behavior. If a change is submitted to a **dot** release
-branch, the dot release branch will be merged into its parent **main**
-release branch.
-
-For example, a pull request is merged into the ``2017.7.2`` release branch.
-Then, the entire ``2017.7.2`` branch is merged-forward into the ``2017.7``
-branch. From there, the merge forward process continues as normal.
-
-The only way in which dot release branches differ from main release branches
-in regard to merge-forwards, is that once a dot release branch is created
-from the main release branch, the dot release branch does not receive merge
-forwards.
-
-.. note::
+When the time comes for a new release which should be every 3 to 4 months
+the release will be cut from the master.
 
-    The merge forward process for dot release branches is one-way:
-    dot release branch --> main release branch.
+To be able to merge code. The code must have well written test.
+Please note you are only expected to write test for what you did not the whole modules or function.
+All tests must also pass.
+The salt stack employee that reviews your pull request might
+request changes or deny the pull request for various reasons.
 
-Closing GitHub issues from commits
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Release Naming Convention
+-------------------------
 
-This "merge-forward" strategy requires that `the magic keywords to close a
-GitHub issue <Closing issues via commit message_>`_ appear in the commit
-message text directly. Only including the text in a pull request will not
-close the issue.
+A new convention will start when Salt releases Salt 3000.
+Every new release name will increment by one ‘Salt last_release_number + 1’.
 
-GitHub will close the referenced issue once the *commit* containing the
-magic text is merged into the default branch (``|repo_primary_branch|``). Any magic text
-input only into the pull request description will not be seen at the
-Git-level when those commits are merged-forward. In other words, only the
-commits are merged-forward and not the pull request text.
+This is very different from past releases which was 'year, month, dot release'.
+For example 2019.2 and 2019.2.3.
```

Comment:
To avoid using an unclear modifying pronoun and to add a comma before "which" for grammar purposes, change to:

This naming convention is very different from past releases, which was 'year, month, dot release'.

Reply:
`YYYY.MM.PATCH` would be more common